lib/js/components/CounterToggle/CounterToggle.vue

```diff
@@ -0,0 +1,367 @@
+<template>
+	<div
+		v-ripple
+		class="counterToggle"
+		:class="[{ '-selected': isSelected, '-disabled': isDisabled }, colorClass]"
+		@click="$emit('click')"
+	>
+		<icon class="counterToggle__icon" :icon="icon" :size="ICON_SIZES.X_SMALL" />
+		<span v-if="counter !== null" class="counterToggle__counter">{{ counter }}</span>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../styles/settings/animations';
+@import '../../../styles/settings/colors/tokens';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/typography';
+
+$counter-toggle-min-width: 46px;
```

Comment:
we should not strict width and height unless it is directly said. `Natural` width and height should come from internal paddings and margins.